### ISSUE TITLE
OS X error

### ISSUE THREAD

**Issue by iShift** [kustomzone/Fuzium#4]
![2017-01-24 21 23 34](https://cloud.githubusercontent.com/assets/426427/22260783/7d2a6b60-e27b-11e6-8b05-2de1fc16e9e1.png)

**Comment by kustomzone** [comment]
Hi, thanks for the screeny. I'm unable to test for Mac.
Tamas has just released Mac and Win app versions. (they open in default browser)

- https://github.com/HelloZeroNet/ZeroNet-mac/tree/dist
- https://github.com/HelloZeroNet/ZeroNet-win/tree/dist

It's taken some time to get Electron installer version running as the node.js paths change.
(i have to have both in **main.js** and check whether it's ```npm start``` or Electron ```.exe```) 
There'll be similar issues with Mac.

I've just made a quick zip release (below) which could be compared with the packs Tamas has made to see the different OS path requirements, but i'm unable to test for Mac atm. Maybe someone else will be able to.

[Win32-zip](https://github.com/kustomzone/Fuzium/releases/tag/v0.0.3-pre-alpha)

For now, i've mentioned 'not running on Mac' in the readme and linked to this issue. Thx.

**Comment by iShift** [comment]
Yep i testes Tamas version (https://github.com/HelloZeroNet/ZeroNet-mac/tree/dist) it works good.
If you want - i can test it on mac
